src/index.html

```diff
@@ -1,329 +1,449 @@
-                              class="pointer" data-pulldown-target="arrow"
-                            >
-                        </div>
-                    </div>
-                    <div id="chart">
-                        <!-- Axis labels -->
-                        <div class="text-s top-label">Progressista</div>
-                        <div class="text-s bottom-label">Conservador</div>
-                        <div class="text-s left-label">Esquerda</div>
-                        <div class="text-s right-label">Direita</div>
-
-                        <!-- Grid lines -->
-                        <div class="main-horizontal dotted-spaced"></div>
-                        <div class="main-vertical left dotted-spaced"></div>
-
-                        <!-- Dotted grid lines -->
-
-                        <!-- Parties positioned on the chart -->
-                        <!-- Partido Socialista - blue circle -->
-                        <div class="party ps" style="left: 40%; top: 45%">PS</div>
-
-                        <!-- Bloco de Esquerda - red star/figure -->
-                        <div class="party be" style="left: 35%; top: 35%">BE</div>
-
-                        <!-- PCP - hammer and sickle -->
-                        <div class="party pcp" style="left: 15%; top: 75%">PCP</div>
-
-                        <!-- Chega - in different positions -->
-                        <div class="party chega" style="left: 70%; top: 20%">CH</div>
-                        <div class="party chega" style="left: 35%; top: 80%">CH</div>
-
-                        <!-- AD - orange -->
-                        <div class="party ad" style="left: 65%; top: 40%">AD</div>
-                        <div class="party ad" style="left: 55%; top: 75%">AD</div>
-                        <div class="party ad" style="left: 20%; top: 85%">AD</div>
-
-                        <!-- Volt - purple -->
-                        <div class="party volt" style="left: 65%; top: 60%">VT</div>
-
-                        <!-- PAN - green -->
-                        <div class="party pan" style="left: 60%; top: 65%">PAN</div>
-
-                        <!-- IL - liberal initiative -->
-                        <div class="party il" style="left: 55%; top: 65%">IL</div>
-
-                        <!-- Extra circles for left side -->
-                        <div class="party be" style="left: 18%; top: 65%">PS</div>
-                        <div class="party be" style="left: 25%; top: 75%">PS</div>
-
-
-                    </div>
-
-                </div>
+          <div class="center mb-2 chart-title">
+            <div class="question-header">
+              <h3>Espectro Político</h3>
             </div>
+          </div>
+
+          <div class="content-box chart">
+            <div class="chart-info" data-controller="pulldown">
+              <p class="text-center text-ss mb-2">
+                <b>Posição Horizontal</b>: eixo <b>Económico</b>. Esquerda (-1)
```

Comment:
It's harder to read when we split these long lines. While having overly long lines is generally better, in this case it makes it harder to follow the content of each line. Can we make these single lines, even if it goes over a character limit?

PS: The [80 character bit is based on 1928's punchcard sizes](https://softwareengineering.stackexchange.com/a/148678).